Utils/addUserHelper.js

```diff
@@ -0,0 +1,23 @@
+const db_Connection = require('../server/database/dbConnection');
+const { hashPassword } = require('../server/helpers/hashHelper');
+
+const usernmae = process.argv[2];
+const password = process.argv[3];
+const id = process.argv[4];
+hashPassword(password, (error, hashedPassword) => {
+  if (error) {
+    console.log('error', error);
+  } else {
+    db_Connection.query(
+      'insert into users (id,full_name,user_name,password,telephone,email,type) values ($1,$2,$3,$4,$5,$6,$7)',
+      [id, usernmae, usernmae, hashedPassword, 242345, 'erge@j.com', 'B'],
+      (err, result) => {
+        if (err) {
+          console.log(err);
+        } else {
+          console.log('success user added', result);
```

Comment:
definitely not a great idea to console log the result of hashing. Instead of console logging, you can send back a message saying `success` so that you have something if you need to check if the operation was successful:

`res.status(200).send('success');`

Reply:
This file won't be run through the server to receive a request, it will through the terminal: 
`node Util/addUserHelper.js username password`